aten/src/ATen/native/mps/operations/UnaryOps.mm

```diff
@@ -272,7 +272,7 @@ static void unary_op(const Tensor& self,
 }
 
 Tensor& angle_out_mps(const Tensor& self, Tensor& output) {
-  TORCH_CHECK(self.scalar_type() != ScalarType::Long, "MPS does not support angle op with int64 input");
+  // TORCH_CHECK(self.scalar_type() != ScalarType::Long, "MPS does not support angle op with int64 input");
```

Comment:
ditto

Reply:
I kept them around, thinking that may be something will fail on MacOS-13, but it did not, so now I can just remove this check